src/components/elements/inputs/boolean-list.js

```diff
@@ -33,7 +41,7 @@ export default class CheckboxList extends Input {
                 checked={isChecked(this.state.value, option)}
```

Comment:
It did work because it handled it as a single-item array with the string as the item but it may be hard to understand the code so that could justify your change.